templates/marketing-location-page/marketing-location-page.js

```diff
@@ -14,8 +14,11 @@ function buildMarketingPage(main) {
   formSection.classList.add('section');
   const formPath = getMetadata('form-path');
   const formStyleClass = getMetadata('form-style');
-  const form = buildBlock('form', { elems: [`<a href="${formPath}"></a>`] });
-  form.classList.add(formStyleClass);
+  const element = { elems: formPath ? [] : [`<a href="${formPath}"></a>`] };
+  const form = buildBlock(formPath ? 'form' : 'get-a-quote-form', element);
```

Comment:
If formpath is provided then we are using it other include get-a-quote-form which have a default form url rite

Reply:
Yes

Reply:
But I was thinking of getting rid of get-a-quote-form block and making its css part of form block. 
@jalagari what do you think??